www/src/components/guidelines/box-with-border.js

```diff
@@ -1,27 +1,26 @@
-import styled from "@emotion/styled"
-import themeGet from "@styled-system/theme-get"
+/** @jsx jsx */
+import { jsx, Box } from "theme-ui"
 
-import { Box } from "./system"
-
-const BoxWithBorder = styled(Box)(
-  props =>
-    props.withBorder && {
-      ":after": {
-        border: `1px solid ${themeGet(`colors.blackFade.10`)(props)}`,
-        borderRadius: `${themeGet(`radii.1`)(props)}`,
-        bottom: 0,
-        content: `" "`,
-        left: 0,
-        position: `absolute`,
-        right: 0,
-        top: 0,
-      },
-    }
-)
-
-BoxWithBorder.defaultProps = {
-  borderRadius: 1,
-  position: `relative`,
+export default function BoxWithBorder({ withBorder, children, ...rest }) {
+  return (
+    <Box
+      sx={{
+        borderRadius: 1,
+        position: `relative`,
+        ":after": withBorder && {
+          content: `" "`,
+          border: `1px solid ${t.colors.blackFade[10]}`,
```

Comment:
Should this be:

```suggestion
          border: t => `1px solid ${t.colors.blackFade[10]}`,
```

Alternately we could do:

```suggestion
          borderWidth: `1px`,
          borderStyle: `solid`,
          borderColor: `blackFade.10`,
```

Reply:
I totally agree with you